influxdb3_write/src/write_buffer/queryable_buffer.rs

```diff
@@ -687,7 +675,7 @@ mod tests {
         // create the initial write with two tags
         let val = WriteValidator::initialize(db.clone(), Arc::clone(&catalog)).unwrap();
         let lp = format!(
-            "foo,t1=a,t2=b f1=1i {}",
+            "foo,t2=a,t1=b f1=1i {}",
```

Comment:
I wanted to change the order here, because the `primary_key` function falls back to lexicographical sorting of tag keys, so verifying `schema.primary_key()` would still pass.